src/views/Home/components/LotteryJackpot.tsx

```diff
@@ -11,17 +11,17 @@ const LotteryJackpot = () => {
   const TranslateString = useI18n()
   const lotteryPrizeAmount = useTotalRewards()
   const balance = getBalanceNumber(lotteryPrizeAmount)
-  const lotteryPrizeAmoutCake = balance.toLocaleString(undefined, {
+  const lotteryPrizeAmountCake = balance.toLocaleString(undefined, {
     maximumFractionDigits: 2,
   })
   const lotteryPrizeAmountBusd = new BigNumber(balance).multipliedBy(usePriceCakeBusd()).toNumber()
 
   return (
     <>
       <Text bold fontSize="24px" style={{ lineHeight: '1.5' }}>
-        {lotteryPrizeAmoutCake} {TranslateString(999, 'CAKE')}
+        {lotteryPrizeAmountCake} {TranslateString(999, 'CAKE')}
```

Comment:
If you can, please update this to a correct translation string

```
TranslateString(999, `${lotteryPrizeAmountCake} CAKE`, { amount: lotteryPrizeAmountCake })
```

Reply:
done